pom.xml

```diff
@@ -157,7 +157,7 @@
       <dependency>
         <groupId>cglib</groupId>
         <artifactId>cglib-nodep</artifactId>
-        <version>3.2.4</version>
+        <version>3.2.10</version>
```

Comment:
This is to avoid errors such as
```
[ERROR] prints a log message on warnings(org.sonatype.nexus.ci.iq.IqPolicyEvaluatorTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException
        at net.sf.cglib.proxy.BridgeMethodResolver.resolveAll(BridgeMethodResolver.java:61)
        ...
```